CSS/style.css

```diff
@@ -114,3 +114,78 @@
 
 }
 
+.second-section {
```

Comment:
![syed1](https://user-images.githubusercontent.com/58483864/219640030-6a7092cc-6edc-4ce1-9519-2c88b27bea2f.png)

- [ ] In larger screens there's a gap because you specify `max-width` to 1440px. In order to center the content you can add `margin: auto` to place `.second-section` to center of its div.